Clarify that some advice does not apply to GovPress (#971)
dxw/playbook: _guides/tech-team-junior-developer-development-plan.md

---

> title: Technology team junior developer development plan

Reviewer: ```suggestion
title: Junior developer development plan (Delivery+)
```

---

> 
> _Please note that this document does not apply to the GovPress team._
> 

Reviewer: ```suggestion

**Please note that this document applies to the Delivery+ team only.**

```
